src/reynir/verbframe.py

```diff
@@ -325,22 +326,27 @@ def create_from_config(cls, s: str) -> None:
 
         args = a[1:]
         # Add to verb database
+        vf = cls(verb, args, prepositions, particle, score)
+        case_key = vf.case_key
         if error:
             # Add this to the error database
             VerbErrors.add_error(verb, args, prepositions, particle, error)
-        # Create a VerbFrame instance
         # Note: In order to parse verbs with wrong arguments,
-        # the frame needs to be present as a regular VerbFrame instance
+        # the frame needs to be present as an extra VerbFrame instance
```

Comment:
Fix indent of this comment (should be indented 1 additional level)

Reply:
Done